src/iqm_client/iqm_client.py

```diff
@@ -334,3 +430,80 @@ def wait_for_results(self, run_id: UUID, timeout_secs: float = DEFAULT_TIMEOUT_S
                 return results
             time.sleep(SECONDS_BETWEEN_CALLS)
         raise APITimeoutError(f"The task didn't finish in {timeout_secs} seconds.")
+
+    def close(self) -> bool:
+        """Terminate session with authentication server.
+        """
+        if self._credentials is None:
+            return False
+
+        if not self._credentials.refresh_token:
+            return False
+
+        url = f'{self._credentials.auth_server_url}/realms/{AUTH_REALM}/protocol/openid-connect/logout'
+        data = AuthRequest(client_id=AUTH_CLIENT_ID, refresh_token=self._credentials.refresh_token)
+        result = requests.post(url, data=data.dict(exclude_none=True))
```

Comment:
```suggestion
        result = requests.post(url, json=data.dict(exclude_none=True))
```
Maybe? I'm not 100% clear on what's the difference between the data and json parameters.

Reply:
Keycloak requires request body to be form data, not raw JSON. Body must be given in `data` argument.